client/my-sites/google-my-business/select-business-type/index.js

```diff
@@ -0,0 +1,92 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import HeaderCake from 'components/header-cake';
+import Card from 'components/card';
+import CTACard from './cta-card';
+import { recordTracksEvent } from 'state/analytics/actions';
+
+class SelectBusinessType extends Component {
+	static propTypes = {
+		translate: PropTypes.func.isRequired,
+		siteId: PropTypes.string.isRequired,
+		recordTracksEvent: PropTypes.func.isRequired,
+	};
+
+	onCreateMyListingClick = () => {
+		this.props.recordTracksEvent( 'calypso_google_my_business_create_my_listing_click' );
+	};
+
+	onOptimizeYourSEOClick = () => {
```

Comment:
Could we use something that better convey what these handlers actually do? I'm thinking of `trackOptimizeYourSEOClicked` for example - similarly to what we did in the nudge.